redis.ts

```diff
@@ -1422,7 +1422,7 @@ class RedisImpl implements Redis {
       args.push("LIMIT", opts.limit.offset, opts.limit.count);
     }
     if (opts?.patterns) {
-      args.push("GET", ...opts.patterns);
+      args.push(...opts.patterns.flatMap((pattern) => (["GET", pattern])));
```

Comment:
Thanks for the PR @whymidnight

Could you run `deno fmt` on this file?

Reply:
got you! amended my commit with update